packages/react-native/template/README.md

```diff
@@ -0,0 +1,76 @@
+This is a [**React Native**](https://reactnative.dev) project bootstrapped with [`npx react-native@latest init`](https://github.com/react-native-community/cli/tree/main#creating-a-new-react-native-project).
```

Comment:
The command here isn't always correct. E.g. you would get the same readme if you initialized a nightly, or any fixed version 0.73+.
```suggestion
This is a new [**React Native**](https://reactnative.dev) project.
```

Reply:
You're right. I also had a thought about it while writing.

Reply:
This is a new [**React Native**](https://reactnative.dev) project, bootstrapped using [`@react-native-community/cli`](https://github.com/react-native-community/cli).